CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -635,7 +635,7 @@ msgstr ""
 
 #. 3.2->settings-schema.json->menu-label->description
 #. 2.6->settings-schema.json->menu-label->description
-msgid "'Menu'-Button label"
+msgid "'Menu'-Button label (shown next to menu icon)"
```

Comment:
@cinnamoner The button that applet places on the panel as a toggle for the menu's visibility